### ISSUE TITLE
Spacing and alignment not working on duplicated symbols in Sketch 50

### ISSUE THREAD

**Issue by thischrisoliver** [DWilliames/paddy-sketch-plugin#113]
#### Expected behavior:

I made a table using a symbol for the cells. Each row of cells is in a group with [0h t] set for spacing and alignment. Normally, if I duplicate a cell by Opt + clicking and dragging, the cells automatically reposition themselves, making room for the new cell.

#### Actual behavior:

After updating to Sketch, the cells aren't automatically repositioning. I've been playing around with some other files, and the repositioning works sporadically on some layers but not always.

![screen shot](https://user-images.githubusercontent.com/22454900/39876418-2005a218-5441-11e8-9093-7c1e512fcba5.png)

**Comment by chsWeb** [comment]
Sometimes you have to select a layer in the group and press the Escape key to fire Paddy into action. I think this is improved in the Paddy 2 implementation.

**Comment by thischrisoliver** [comment]
Yes, I know I have to do that sometimes to get Paddy to work. Since the update, though, I've done that numerous times, but Paddy still won't realign the layers.
